spec/symengine_spec.rb

```diff
@@ -18,5 +18,12 @@
     it { is_expected.to be_a SymEngine::Rational }
     its(:to_s) { is_expected.to eq '1/3' }
   end
+  
+  describe 'parse' do
+    subject { SymEngine::parse('123 + 321') }
+    
+    it { is_expected.to be_a SymEngine::Integer }
+    it { is_expected.to eq 444 }
```

Comment:
That's a quite nice way to make it clear! 

I'll keep this as it is, because week 9 is just one week away from now, probably I will start working on it even earlier.